tests/modules/programs/firefox/profiles/search/default.nix

```diff
@@ -53,9 +53,9 @@ in {
             "NixOS Wiki" = {
               urls = [{
                 template =
-                  "https://wiki.nixos.org/index.php?search={searchTerms}";
+                  "https://wiki.nixos.org/w/index.php?search={searchTerms}";
               }];
-              iconUpdateURL = "https://wiki.nixos.org/favicon.png";
+              iconMapObj."16" = "https://wiki.nixos.org/favicon.png";
```

Comment:
this was inherited from the previous code, but while it's being updated anyways:
this url is actually a 404, it's `favicon.ico`

Reply:
oh yep thanks for catching that!